.changeset/weak-bananas-deliver.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/plugin-pagerduty': patch
```

Comment:
This should be a minor change, as we're technically breaking the `fromConfig` method in the `api-report.md`

Reply:
Makes sense! I was conflicted about this initially since I would expect the client is setup automatically and wouldn't require manual intervention, but thats likely an incorrect generalization.

Updated to `minor`!